gflownet/envs/cube.py

```diff
@@ -563,16 +595,50 @@ def absolute_to_relative_increments(
 
         r = (a - m) / (x - m)
         """
-        max_val = torch.full_like(states, max_val)
+        min_increments = torch.full_like(
+            increments_abs, self.min_incr, dtype=self.float, device=self.device
+        )
         if is_backward:
-            increments_rel = (increments_abs - min_increments) / (
-                states - min_increments
-            )
+            return (increments_abs - min_increments) / (states - min_increments)
         else:
-            increments_rel = (increments_abs - min_increments) / (
-                max_val - states - min_increments
-            )
-        return increments_rel
+            return (increments_abs - min_increments) / (1.0 - states - min_increments)
+
+    @staticmethod
+    def _get_beta_params_from_mean_variance(
+        mean: TensorType["n_states", "n_dim_x_n_comp"],
+        variance: TensorType["n_states", "n_dim_x_n_comp"],
+    ) -> Tuple[
+        TensorType["n_states", "n_dim_x_n_comp"],
+        TensorType["n_states", "n_dim_x_n_comp"],
+    ]:
+        """
+        Calculates the alpha and beta parameters of a Beta distribution from the mean
+        and variance.
+
+        The method operates on tensors containing a batch of means and variances.
+
+        Args
+        ----
+        mean : tensor
+            A batch of means.
+
+        variance : tensor
+            A batch of variances.
+
+        Returns
+        -------
+        alpha : tensor
+            The alpha parameters for the Beta distributions as a function of the mean
+            and variances.
+
+        beta : tensor
+            The beta parameters for the Beta distributions as a function of the mean
+            and variances.
+        """
+        one_minus_mean = 1.0 - mean
+        beta = one_minus_mean * (mean * one_minus_mean - variance) / variance
+        alpha = (mean * beta) / one_minus_mean
```

Comment:
So I didn't get into too much detail on why this is the case, but the wikipedia page on beta distribution states that this is the case if `variance < mean * (1 - mean)` - assuming that's true, perhaps add some assert to validate this here?

Reply:
I will invite @AlexandraVolokhova to this party.

Reply:
Since it is unused I will not address this here. I have opened an issue as a reminder instead.